api/app/settings/logger.py

```diff
@@ -0,0 +1,22 @@
+import json
+import logging
+
+
+class JsonFormatter(logging.Formatter):
+    """Custom formatter for json logs."""
+
+    def format(self, record):
+        try:
+            log_message = record.msg.replace('"', "'").replace("%s", "{}")
```

Comment:
I'd love to see a comment here explaining why we're having to do these replaces. I think you explained it in the description of this PR so perhaps we can just add that detail here too?

Reply:
json.dumps seems to be handling the quote character replacement fine after some testing, so it was removed.  Comments were added for clarity around the issue with the logging module not passing arguments  correctly with legacy string-formatting tokens.